app/controllers/users/two_step_verifications_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Users::TwoStepVerificationsController < ApplicationController
+  def new
+    redirect_to two_step_verification_path if current_user.two_factor_auth_enabled?
```

Comment:
Yes, now I get it. If TFA is enabled already, you want to show that and give the option to disable it. 👍